### PR TITLE
Natto book

### PR DESCRIPTION
**Pull request by daizu27**
@CamDavidsonPilon , doing some promo for the book, and that should be it! Gonna take a short break after. 

I added the .md file and photos, not sure what else needs to be done.

### CODE REVIEW

**Comment by CamDavidsonPilon** [comment]
> Gonna take a short break after.

👍 rest is important

**Comment by CamDavidsonPilon** [comment]
Sorry there was a problem deploying, I'll look into it and get it online

**Comment by CamDavidsonPilon** [comment]
Online now :)

https://controlledmold.com/natto/

**Comment by daizu27** [comment]
thanks! looks like the images are broken. does it have to do with the image size? my JPGs out of camera are kind of big

**Comment by CamDavidsonPilon** [comment]
It's the same errors as I fixed [here](https://github.com/ControlledMold/ControlledMold.github.io/commit/22058855c870974172e2d2c2601d51f3dc739568): `jpg` -> `JPG`

File: _posts/2023-04-11 natto.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+### There were no detailed books on Natto
+
+In 2022, if you searched 'natto book' on Amazon.com, the first result is a self-help / insomia book: something about a problem getting stickier the more you stir it. I'm sure it's a great book, but... come on. 
+
+Aside from that, there are a few decent home fermentation books with passing mentions about natto. Kristen and Christopher Shockey's book is great, and so is Sandor Katz's. Natto chapters also show up in a few textbooks on fermented foods, but none of them have the sort of detail I'm looking for. 
+
+What kind of book did I want? Something like this:
+
+![](/assets/images/natto-book/kojistudies_book.jpg){:width="550" .center}
```
Comment (CamDavidsonPilon) [inline _posts/2023-04-11 natto.md:22]: @daizu27 it's the name of the path, too: `natto-book` vs `natto_book`. I'll make the fix and deploy

**Pull request merged**